LibreNMS/Modules/Os.php

```diff
@@ -129,6 +129,7 @@ private function handleChanges(\LibreNMS\OS $os): void
 
         echo trans('device.attributes.location') . ': ' . optional($device->location)->display() . PHP_EOL;
         foreach (['hardware', 'version', 'features', 'serial'] as $attribute) {
+            $device->$attribute = trim($device->$attribute);
```

Comment:
This changes all nulls to ''

Reply:
I'm don't know if that is good or bad. Thoughts?

Reply:
It does cause an deprecation, fixed it.